homeassistant/components/v2c/const.py

```diff
@@ -0,0 +1,43 @@
+"""Constants for the Trydan integration."""
+DOMAIN = "v2c"
+
+CHARGE_STATES = [
```

Comment:
These should be translatable https://developers.home-assistant.io/blog/2022/12/01/entity_translations?_highlight=state&_highlight=trans